### PR TITLE
Refactor Assignment Members Api according to LMS Assignment

### PR DESCRIPTION
**Pull request by Muhannado**

### CODE REVIEW

File: AssignmentManager/AssignmentMembersApi.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            {
+                return new BadRequestErrorMessageResult($"Invalid assignment Guid {assignmentGuid}");
+            }
+            var lmsAssignment = await GetLmsAssignment(assignments, queryQuid);
+            if (lmsAssignment == null)
+            {
+                return new NotFoundObjectResult($"Assignment with Guid {assignmentGuid} was not found");
+            }
+            var membershipsManager = new LmsMemberships();
+            var members = await membershipsManager.GetMemberships(lmsAssignment.CustomContextMembershipsUrl,
```
Comment (amichaiv) [inline AssignmentManager/AssignmentMembersApi.cs]: It's should call membershipsAccessor not membershipsManager

File: AssignmentManager/LmsAssignment.cs [hunk 2]
```diff
@@ -2,10 +2,23 @@
 
 namespace AssignmentsManager
 {
-    class LmsAssignment : TableEntity
+    internal class LmsAssignment : Assignment
```
Comment (amichaiv) [inline AssignmentManager/LmsAssignment.cs]: Why to inerrant from Assignment?

File: AssignmentManager/AssignmentMembersApi.cs (hunk 1 above)
Comment (Muhannado) [reply AssignmentManager/AssignmentMembersApi.cs]: Is it only about naming? call it lmsMembershipsAccessor instead

File: AssignmentManager/LmsAssignment.cs (hunk 2 above)
Comment (Muhannado) [reply AssignmentManager/LmsAssignment.cs]: So I won't put the properties again, but yes I just paid close attention, no need for it

**Pull request merged**